### PR TITLE
ContextualMenu accessibility: Fix high contrast style for submenu icon

### PR DESCRIPTION
**Pull request by erichdev**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Fix high contrast style for ContextualMenu submenu icon. In "high contrast white", the icon currently does not have enough contrast with background on focus/hover.

**Before**

high contrast

**After**

high contrast fix

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts [hunk 1]
```diff
@@ -24,6 +24,11 @@ const getItemHighContrastStyles = memoizeFunction(
           borderColor: 'Highlight',
           color: 'HighlightText',
           MsHighContrastAdjust: 'none',
+          selectors: {
```
Comment (xugao) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts:27]: can you add this style to `subMenuIcon: { ... }` block below instead targeting the icon using className?

Comment (erichdev) [reply packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts]: Thanks @xugao I've moved the change down to `subMenuIcon`

Comment (xugao) [reply packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts]: Thanks!

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts [hunk 2]
```diff
@@ -225,6 +225,9 @@ export const getMenuItemStyles = memoizeFunction(
           [MediumScreenSelector]: {
             fontSize: IconFontSizes.medium, // 16px
           },
+          [HighContrastSelector]: {
+            color: 'HighlightText',
```
Comment (xugao) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts:229]: This should be WindowText color

Comment (erichdev) [reply packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts:229]: @xugao  The background in high contrast is `Highlight`, so should the text color be `HighlightText`, similar to how it is for the root text and for `iconColor`? 

I tested locally setting it to `WindowText`, and in high contrast white, the icon looks similar to the "Before" picture in the description, where there's not much contrast with the background.

Comment (xugao) [reply packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts:229]: Ah you are right.

Comment (xugao) [reply packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts:229]: I merged the change for you:)

**Pull request merged**